pyproject.toml

```diff
@@ -8,6 +8,7 @@ authors = []
 python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
+requests = "^2.25.1"
```

Comment:
It looks like you haven't committed the changes to your `poetry.lock` file. 

Try doing:
```bash
rm -r .venv/
poetry install
```
I expect you'll get an error like I did running `poetry install`. Try running `poetry update` and then committing the changes to `poetry.lock`